dask_ml/linear_model/glm.py

```diff
@@ -14,8 +14,6 @@
 )
 from sklearn.base import BaseEstimator
 
-# Register multiple dispatch
-from . import utils  # noqa
```

Comment:
Note: this was the only change that was made by me and not by just running `isort`. `black` and `isort` both wanted to change this line in incompatible ways. However, it appears that `utils` wasn't being used in this file, so I just removed the import altogether.